### ISSUE TITLE
[Question] Does this project use websocket?

### ISSUE THREAD

**Issue by ne0YT** [boschkundendienst/guacamole-docker-compose#83]
hey there

does this project use websocket for client connections?

**Comment by craeckor** [comment]
Yes, guacamole uses websocket as default connection, but if websocket is not available somehow, it uses a backup solution with https packets but the performance is really shitty on this and you have intensive packet loss which can cause graphics errors, also your server will recieve basically a big https flood, if websocket doesn't work. (If you have multiple active clients at the same time)
